main.py

```diff
@@ -3864,15 +3865,30 @@ async def on_submit(self, interaction: discord.Interaction):
                 await update_trade_embed(interaction)
                 return
 
-            if self.cattype.value not in cattypes:
+            # lowercase input to normalise
+            lc_input = self.cattype.value.lower()
+
+            # check for edge case cat names and normalise where necessary
+            if lc_input == "egirl":
+                cname = "eGirl"
+            elif lc_input == "8bit":
+                cname = "8bit"
+            elif lc_input == "thetrashcell":
+                cname = "TheTrashCell"
+            else:
+                # normalise input to title case
+                cname = self.cattype.value.title()            
```

Comment:
im not a big fan of this at all, there are much less cursed ways of doing this
the thing im thinking of is just iterating over cattypes and seeing if cattype.lower() == lc_input

Reply:
iterating over a few strings is fine, if you really want to you can build a {cattype.lower(): cattype} dictionary at boot

i mean comparing all of them, and is lower() matches, save that and break